portfolio/src/main/webapp/game.js

```diff
@@ -9,7 +9,23 @@ class Board {
       [0, 0, 0]
     ];
     this.drawGrid();
-    var red = true; 
+    this.red = true; 
+    this.numTiles = 0;
+  }
+
+  /* 
+  * Initalizes game
+  */
+  init() {
```

Comment:
Constructor could call `init` so that there's no duplication.

Reply:
Got it.